modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -11,7 +11,7 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Module houses classes responsible for storing an axis partition and applying a function to it."""
+"""Module houses classes responsible for storing a virtual partition and applying a function to it."""
```

Comment:
Should we rename `PandasOnDaskDataframeAxisPartition` to `PandasOnDaskDataframeVirtualPartition`?

Reply:
I think we can rename this class when we make the changes as in `PandasOnRayVirtualPartition`. Right now it only supports `AxisPartition` behaviors, so the name can probably stay.

Reply:
Maybe it is worth reverting these changes?

Reply:
decided to leave as it is.